resources/assets/components/Post/index.js

```diff
@@ -105,7 +108,7 @@ class Post extends React.Component {
               : null}
 
             <div className="container -padded">
-              <TextBlock title="Photo Caption" content={displayCaption(post)} />
+              <TextBlock title="Text" content={displayCaption(post)} />
```

Comment:
Could we call this `Text` if it is of `type=text` and "photo caption" if it is of `type=photo`?

Reply:
ooo that is a good idea!